indeex.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="stylesheet.css">
+    <header>
+        <!-- section which is not needed -->
+    <div class="logo"><!-- logo will be put here --></div>
+    <nav> 
+        <ul>
+            <li><!--Home --></li>
+            <li><!--About --></li>
+            <li><!--Services --></li>
+            <li><!-- Gallery --></li>
+            <li><!--Blogs --></li>
+            <li><!-- Contact--></li>
+
+        </ul>
+    </nav> 
+    <h3><a href="#">Login</a></h3>
+    </header>
+</head>
+<body>
+<!-- container practice -->
+
+<div class="main-container">
+
+    <div class="first-container">
+            <div class="item1">
+                <h2>About </h2>
+                    <p>its Mishal firdous here, i am a software engineer, Previsiosly having one year CMS expertices in wordpress,
+                         wix and shopify.Which plays a critical role in the technology industry. Through these content management 
+                         system design would more eye cautching although responsible for designing, creating, and maintaining software
+                          that meets the needs of businesses and users in various domains, including finance, healthcare, e-commerce, 
+                          entertainment, and many others. Software engineers work with programming languages, algorithms, and software 
+                          development methodologies to build applications and systems that drive modern society.</p>
+            </div>
+            <div class="item2">
+                <h2>Experince</h2>
+            </div>
+    </div>
+
+    <div class="sec-container">
+       
+        <div class="item3">
+            <img src="c:\Users\MishalFirdous\Downloads\WhatsApp Image 2023-10-20 at 3.42.56 AM.jpeg">
```

Comment:
This image will only be shown while the code is run on your machine. Keep the image in the assets folder and use it here.

Reply:
resolved!